db/seeds.d/20-permissions.rb

```diff
@@ -7,3 +7,15 @@
     filter.filterings.build { |f| f.permission = view_permission }
   end
 end
+
+site_manager_role = Role.find_by(name: 'Site manager')
```

Comment:
Can't we just add the 'Site manager' to `AUTO_EXTENDED_ROLES` and edit it in the engine file?
 https://github.com/theforeman/foreman/blob/ef2ff06831ae7efcdab247b13c9ff642c6d56688/app/registries/foreman/plugin/rbac_support.rb#L5

Reply:
We definitely could, do you know if there's a reason why it currently is excluded from AUTO_EXTENDED_ROLES?

Reply:
Nevermind, did that

Reply:
Because it was added later and noone actually thought of adding it :shrug: